simulation-ros/src/README.md

```diff
@@ -0,0 +1,18 @@
+# The Src folder
+
+
+All the source code files for all packages are in this folder. 
+
+The three foldes here are meta-packages.
+### 1.vrep_ros_interface
+There is no need for the developer to check this package. We have installed it in the previous configuration.
```

Comment:
as a matter of fact, have you figured out how to configure it to connect to the ROS Master which is not on localhost?

Reply:
Like [this](http://wiki.ros.org/ROS/Tutorials/MultipleMachines) ?

Reply:
yes, but I specifically mean the ROS plugin for VREP. Last time I tried it failed to start (ie VREP starts w/o a ROS plugin active) if master was not running on localhost:11311